pkg/collector/metrics.go

```diff
@@ -43,10 +45,12 @@ var (
 	availableCgroupMetrics  []string = cgroup.GetAvailableCgroupMetrics()
 	availableKubeletMetrics []string = podlister.GetAvailableKubeletMetrics()
 	// TO-DO: merge to cgroup stat and remove hard-code metric list
-	iostatMetrics []string = []string{ByteReadLabel, ByteWriteLabel}
-	uintFeatures  []string = getUIntFeatures()
-	features      []string = append(FloatFeatures, uintFeatures...)
-	metricNames   []string = getEstimatorMetrics()
+	iostatMetrics  []string = []string{ByteReadLabel, ByteWriteLabel}
+	uintFeatures   []string = getUIntFeatures()
```

Comment:
The concept of "[feature](https://en.wikipedia.org/wiki/Feature_(machine_learning))" is very much related to machine learning....
Where `feature` is an individual measurable property or characteristic of a phenomenon...

So, in the end, what is the difference between Metrics and Features?

I think we should only use metric, having both makes the code a little confusing...

I understand that the the Linear Model and the other machine learning packages can have the concept of feature, where both numerical metrics and string metadata are used...
But the metric package should only have numerical metrics and metadata...

Reply:
But of course, we can update it later in another PR

Reply:
Exactly, when I named, I think the metrics are for reader and collector but the features are more for modelling. 
We can move the feature parts to model package later.